### PR TITLE
[WIP] poc: cluster singleton for PO

### PR DESCRIPTION
**Pull request by tylerslaton**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -170,7 +171,7 @@ CERT_MGR_VERSION=v1.7.1
 rukpak:
 	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MGR_VERSION)/cert-manager.yaml
 	kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
-	kubectl apply -f https://github.com/timflannagan/rukpak/releases/download/$(RUKPAK_VERSION)/rukpak.yaml
+	kubectl apply -f https://github.com/operator-framework/rukpak/releases/download/$(RUKPAK_VERSION)/rukpak.yaml
```
Comment (timflannagan) [inline Makefile:174]: Nice catch: can we fix this in the main branch first?

File: controllers/platformoperator_controller.go [hunk 2]
```diff
@@ -91,20 +91,25 @@ func (r *PlatformOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
...
-		})
-		return ctrl.Result{}, err
+	// TODO: Attempt a dry run before applying for real to make sure resources
+	// 		 all get created together.
+	for _, desiredBundle := range desiredBundles {
+		if err := r.Applier.Apply(ctx, po, desiredBundle); err != nil {
+			meta.SetStatusCondition(&po.Status.Conditions, metav1.Condition{
+				Type:    platformv1alpha1.TypeApplied,
+				Status:  metav1.ConditionFalse,
+				Reason:  platformv1alpha1.ReasonApplySuccessful,
```
Comment (exdx) [inline controllers/platformoperator_controller.go:101]: nit: should this be `ReasonApplyUnsuccessful` instead if the application failed?

File: internal/sourcer/registry_sourcer.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	channelName = "4.12"
+	channelName = "beta"
```
Comment (exdx) [inline internal/sourcer/registry_sourcer.go:16]: q: why the change here?

**Review by exdx — COMMENTED** [review]
I like the UX of the singleton approach, nice. It does introduce some more complexities on the development side though.

File: Makefile (hunk 1 above)
Comment (exdx) [reply Makefile:174]: Was turning CI red so I just put it up in #38

**Comment by tylerslaton** [comment]
Closing due to age.

**Pull request closed without merging**